Wireframe/css/style.css

```diff
@@ -0,0 +1,160 @@
+@import url('https://fonts.googleapis.com/css2?family=Arsenal&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+:root {
+    --dark-blue: #071221;
+    --white: #ffff;
+
+}
+
+body {
+    margin: 0;
+    min-height: 100vh;
+
+}
+
+body,
+button {
+    font-family: 'Arsenal', sans-serif;
+    color: var(--grey-dark);
+    font-size: 1rem;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+p {
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+a {
+    text-decoration: none;
+}
+
+.header,
+.content,
+.footer {
+    padding: 0 1.5rem;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    text-align: center;
+
+}
+
+.title {
+    margin: 2rem 0;
+    font-size: 2.5rem;
+}
+
+.hero {
+    height: 25rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+    margin-bottom: 5rem;
+}
+
+.hero__card {
+    height: 14rem;
```

Comment:
What caused you to decide to set this at 14rem? Is it a bit of a magic number? 

https://css-tricks.com/magic-numbers-in-css/

Reply:
I think, it is. How to avoid using magic number?

Reply:
Do you need a height? What is the set height doing for you? What job is it doing in your code, I mean. If you can identify that we will get clearer on what to do instead.